server/server.go

```diff
@@ -45,6 +47,12 @@ func New(c *Config) (*Server, error) {
 		Pretty: c.Logging.Text,
 	})
 
+	if v, ok := os.LookupEnv("BULLDOZER_PORT"); ok {
+		if i, err := strconv.Atoi(v); err == nil {
+			c.Server.Port = i
+		}
+	}
```

Comment:
Instead of only supporting `BULLDOZER_PORT` here, I'd like to do something that will support additional variables as well:

In the `ParseConfig` function in `server/config.go` somewhere around L68, you can add a call to `c.Server.SetValuesFromEnv("BULLDOZER_")`. This will parse the `BULLDOZER_PORT` variable, but also add support for configuring the listen address, TLS options, and other properties.

Reply:
That sounds awesome. I will try to get to this in the coming days. Thanks so much for your help and support!

Reply:
@bluekeyes sorry for taking so long to get back to you on this! I looked into adding `c.Server.SetValuesFromEnv`, but it appears that the `SetValuesFromEnv` method wasn't added until a later version of `go-baseapp` than what this project is using. I submitted https://github.com/palantir/bulldozer/pull/228 to hopefully unblock us. Thanks again!

Reply:
Thanks for doing the update as a separate PR. I've merged that, so you should be good to update this.

Reply:
@bluekeyes of course!  I think this should be all set now. I rebased for a clean git history and updated the PR description. Thanks again!